### DISCUSSION TITLE
Welcome to CloudFlare Queue Consumer!

### DISCUSSION THREAD

**Discussion by nicholasgriffintn** [bbc/cloudflare-queue-consumer#5] [Announcements]
Welcome to this new package for consuming queue messages from CloudFlare Queues!

Inspired by the work that we did with SQS Consumer, we have decided to expand to a new package designed solely for use with the CloudFlare Queues product.

If you have any questions or feature suggestions, please feel free to open a discussion!

**Comment by switz** [comment]
Let me know what the outstanding issues are and I can try and tackle some

**Reply by nicholasgriffintn** [reply]
Thanks! I've got some basic scratches in the codebase at the moment, there are a few TODOs noted around the place.

Polling seems to be working well, having some issues with acknowledgement.

Update: CloudFlare have responded to a ticket about acknowledgment issues, seems to be an issue with the API: https://community.cloudflare.com/t/cloudflare-queue-acknowledgement-doesnt-seem-to-remove/635212/4?u=nicholasgriffin
